stage0/run.go

```diff
@@ -250,6 +251,36 @@ func setupAppImage(cfg Config, img types.Hash, cdir string) (*schema.ImageManife
 		return nil, fmt.Errorf("error unmarshaling app manifest: %v", err)
 	}
 
+	// Create a minimal set of device nodes
+	// https://github.com/appc/spec/issues/127
+	//
+	// Also inspired from: systemd/src/nspawn/nspawn.c copy_devnodes():
+	// http://cgit.freedesktop.org/systemd/systemd/tree/src/nspawn/nspawn.c#n1345
+	devnodes := []string{"null", "zero", "full", "random", "urandom", "tty", "net/tun"}
+	for _, dev := range devnodes {
+		from := path.Join("/dev/", dev)
+		to := path.Join(ad, "rootfs/dev", dev)
+
+		fi, err := os.Stat(from)
+		if err != nil {
+			return nil, fmt.Errorf("Cannot stat %s: %v", from, err)
```

Comment:
It should not return an error if the kernel is compiled without tun support and /dev/net/tun is missing.